nengo_bones/templates/pyproject.toml.template

```diff
@@ -0,0 +1,16 @@
+# Automatically generated by nengo-bones, do not edit this file directly
+
+[build-system]
+requires = ["setuptools", "wheel"]
+
+[tool.black]
+target-version = ['py35']
+exclude = '''
+(
+      docs/conf.py
+    | setup.py
```

Comment:
Random thought, would it be possible to run our templates through black during the rendering process (so our templated files would always be black (and probably pylint/flake8) compliant)?

Reply:
Yeah, I don't see why not! in the worst case we can open a subprocess, but I bet there's a programmatic way to do it by import `black`. It would mean another dependency, but I'm fine with that. I'll see how easy it is and push a commit with it that we can skip if we don't like it in the end.

Reply:
Pushed that commit, I think it worked out pretty cleanly.